acoustics/_signal.pyx

```diff
@@ -130,6 +130,22 @@ class Signal(numpy.ndarray):
         gain = decibel - other.leq()
         return self.gain(gain, inplace=inplace)
 
+    def calibrate_peak(self, decibel, inplace=False):
+        """Calibrate signal knowing its SPL for peak FS.
+
+        :param decibel: Peak FS value to calibrate to.
+        :param inplace: Whether to perform inplace or not.
+        :returns: Calibrated signal.
+        :rtype: :class:`Signal`
+
+        Values of `decibel` are broadcasted. To set a value per channel, use `decibel[...,None]`.
+        """
+        # Calculate the gain so that samples represent the pressure values
+        # We know that peak value equals 1. 1 Pa is roughly 94 dB SPL.
+        # In that case, we have to boost the signal by decibel - ~94.
+        gain = decibel * np.ones(self.shape) - 20*np.log10(1/2e-5)
```

Comment:
Even though it's trivial, use the `sound_pressure_level` function from `iso_tr_25417_2007`. I prefer to have as few as possible actual implementation in this module.

Reply:
Makes sense. Will do!

Reply:
@FRidh Done.